src/core/stream.c

```diff
@@ -659,6 +661,38 @@ QuicStreamParamSet(
         break;
     }
 
+   case QUIC_PARAM_STREAM_RELIABLE_OFFSET_SEND:
+
+        if (BufferLength != sizeof(uint64_t) || Buffer == NULL) {
```

Comment:
I just noticed that `QUIC_PARAM_STREAM_PRIORITY` above doesn't check for NULL. Can you fix that please?

Reply:
For sure!